sktime/forecasting/compose/_pipeline.py

```diff
@@ -204,11 +382,11 @@ def _update(self, y, X=None, update_params=True):
 
         for step_idx, name, transformer in self._iter_transformers():
             if hasattr(transformer, "update"):
```

Comment:
Mh, shouldn't all transformers have an `update`? By default they should fall back to re-fitting similar to what we do for forecasters.

Reply:
this is not defined in the parent classes, but would be good to have it proably. I there already an issue for refactoring transformers where we can note this down? Also the `y = y.copy()` should be moved to parent.

Reply:
Yes let's do this in a separate issue